SPECS/libkcapi/libkcapi.spec

```diff
@@ -0,0 +1,567 @@
+%global vmajor            1
+%global vminor            2
+%global vpatch            0
+# This package needs at least Linux Kernel v4.10.0.
+%global min_kernel_ver    4.10.0
+# Do we need to tweak sysctl.d? In newer versions of the Linux
+# Kernel the default ancillary buffer size is set high enough.
+# TODO: Adapt this when the patch for net/core/sock.c is merged.
+%if %{lua:print(rpm.vercmp('99.0.0', posix.uname('%r')));} >= 0
+%global with_sysctl_tweak 1
+%else
+%global with_sysctl_tweak 0
+%endif
+# Lowest limit to run the testsuite.  If we cannot obtain this
+# value, we asume the testsuite cannot be run.
+%global test_optmem_max   %(cat /proc/sys/net/core/optmem_max || echo 0)
+# For picking patches from upstream commits or pull requests.
+%global giturl            https://github.com/smuellerDD/%{name}
+%global apps_hmaccalc sha1hmac sha224hmac sha256hmac sha384hmac sha512hmac
+%global apps_fipscheck sha1sum sha224sum sha256sum sha384sum sha512sum md5sum fipscheck fipshmac
+# On old kernels use mock hashers implemented via openssl
+%if %{lua:print(rpm.vercmp(posix.uname('%r'), '3.19'));} >= 0
+%global sha512hmac bin/kcapi-hasher -n sha512hmac
+%global fipshmac   bin/kcapi-hasher -n fipshmac
+%else
+%global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
+%global fipshmac   bash %{_sourcedir}/fipshmac-openssl.sh
+%endif
+# Add generation of HMAC checksums of the final stripped
+# binaries.  %%define with lazy globbing is used here
+# intentionally, because using %%global does not work.
+%define __spec_install_post                                      \
+%{?__debug_package:%{__debug_install_post}}                      \
+%{__arch_install_post}                                           \
+%__os_install_post                                             \
+bin_path=%{buildroot}%{_bindir}                                  \
+lib_path=%{buildroot}/%{_lib}                                    \
+for app in %{apps_hmaccalc}; do                                  \
+  test -e "$bin_path"/$app || continue                           \
+  { %{sha512hmac} "$bin_path"/$app || exit 1; }                    \\\
+    | cut -f 1 -d ' ' >"$lib_path"/hmaccalc/$app.hmac            \
+done                                                             \
+for app in %{apps_fipscheck}; do                                 \
+  test -e "$bin_path"/$app || continue                           \
+  %{fipshmac} -d "$lib_path"/fipscheck "$bin_path"/$app || exit 1  \
+done                                                             \
+%{fipshmac} -d "$lib_path"/fipscheck                               \\\
+  "$lib_path"/libkcapi.so.%{version} || exit 1                   \
+ln -s libkcapi.so.%{version}.hmac                            \\\
+  "$lib_path"/fipscheck/libkcapi.so.%{vmajor}.hmac               \
+%{nil}
+# Do we build the replacements packages?
+%bcond_with replace_coreutils
+%bcond_without replace_fipscheck
+%bcond_without replace_hmaccalc
+%bcond_without test_package
```

Comment:
We're not running our builds using the `--with-*` or `--without-*` switches. I'd suggest removing all `%bcond_*` macros and cleaning-up the spec accordingly to limit the noise.